.github/workflows/ci.yml

```diff
@@ -358,7 +358,10 @@ jobs:
       - prepare-backport-packages
       - build-prod-image
       - docs
-    if: github.ref == 'refs/heads/master' && github.event_name != 'schedule'
+    if: |
+      (github.ref == 'refs/heads/master' ||
+      github.ref == 'refs/heads/v1-10-test' ) &&
```

Comment:
v1-10-stable here too please

Reply:
lol @ashb I just removed it based on @potiuk's suggestion

Reply:
https://github.com/apache/airflow/pull/8955#pullrequestreview-416431693

Reply:
wait, just saw the comment. reading that

Reply:
It would appear that we now have what is called a mexican standoff...

Reply:
ah since they user the same image that's cool.

Reply:
https://youtu.be/BYUzax_34Iw?t=229